Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/RefreshTokenResponse.java

```diff
@@ -0,0 +1,73 @@
+/*-
+ * #%L
+ * This file is part of "Apromore Core".
+ * %%
+ * Copyright (C) 2018 - 2021 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package org.apromore.portal;
+
+public class RefreshTokenResponse {
+    private String authHeader;
+    private String signedAuthHeader;
+    private boolean wasRefreshed;
+
+    public RefreshTokenResponse() {
```

Comment:
Can use lombok here and remove all code.

Reply:
Utilised Lombok, to remove boilerplate code.

```
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class RefreshTokenResponse {

    @Getter @Setter private String authHeader;
    @Getter @Setter private String signedAuthHeader;
    @Getter @Setter private boolean wasRefreshed;
}
```